api/src/main/java/org/pmiops/workbench/rdr/RDRConfig.java

```diff
@@ -0,0 +1,44 @@
+package org.pmiops.workbench.rdr;
```

Comment:
Package naming thought: in line with my other comments strewn about, I think we should treat this package as the "RDR export" package. This might motivate us to rename the package to "rdrexport", and use this as a base name for all other classes: RdrExportService, RdrExportConfig, etc.

I don't have a super-strong feeling here, but I suspect there will be less overall cognitive overhead if we stick to one thing, and I think "RDR export" will capture the gist of it pretty well.